js/components/controlBar.js

```diff
@@ -230,13 +238,11 @@ var ControlBar = React.createClass({
           onMouseOver={this.highlight} onMouseOut={this.removeHighlight}/>
       </button>,
 
-      "live": <div className="oo-live oo-control-bar-item" key="live">
-        <div className="oo-live-indicator">
-          <div className="oo-live-circle"></div>
-          <span className="oo-live-text"> {liveText}</span>
-        </div>
-      </div>,
-
+      "live": <button className="oo-control-bar-item" onClick={this.handleLiveClick} key="live">
```

Comment:
removing 'oo-live' breaks unit tests, also I think QA is using these unique class names in automation to check for elements present